### DISCUSSION TITLE
Nerd Fonts Noto Sans Regular Font Family in CSS

### DISCUSSION THREAD

**Discussion by Kruziikrel13** [ryanoasis/nerd-fonts#889] [Q&A]
Hi, I was wondering if there was documentation the font families for the nerd fonts.

I have installed the Patched Noto Fonts successfully. However getting the CSS font family to use it in waybar is not working as I am unsure of the font family. I was trying to determine the font family for Noto Sans Nerd Font Regular or something along those lines however I cannot find any documentation on the font families.

**Comment by Finii** [comment]
`NotoSans Nerd Font`

![image](https://user-images.githubusercontent.com/16012374/191754999-78a9a2c7-4fd0-4e62-a66b-595cb6e57343.png)
